lib/fluent/test/helpers.rb

```diff
@@ -25,9 +25,13 @@ def config_element(name = 'test', argument = '', params = {}, elements = [])
         Fluent::Config::Element.new(name, argument, params, elements)
       end
 
-      def event_time(str=nil)
+      def event_time(str=nil, format=nil)
```

Comment:
Can you use keyword argument `event_time(str=nil, format: nil)` instead of optional argument?